packages/subcommands/src/lib/structures/Command.ts

```diff
@@ -0,0 +1,19 @@
+import { Args, Command as BaseCommand, CommandOptions } from '@sapphire/framework';
+import type { PieceContext } from '@sapphire/pieces';
+
+export abstract class Command<T = Args> extends BaseCommand<T> {
+	public subcommands: string[];
+
+	public parentName?: string;
+
+	/**
+	 * @since 1.0.0
+	 * @param context The context.
+	 * @param options Optional Command settings.
+	 */
+	protected constructor(context: PieceContext, { name, ...options }: CommandOptions = {}) {
+		const pieceName = (name ?? context.name).toLowerCase();
+		super(context, { ...options, name: pieceName });
+		this.subcommands = options?.subcommands ?? [];
```

Comment:
The optional chaining here is unnecessary; `options.subcommands ?? []` should suffice.